opentreemap/treemap/js/src/searchBar.js

```diff
@@ -160,8 +160,14 @@ function updateActiveSearchIndicators(search) {
         } else {
             var featureName = key.split('.')[0],
                 featureCategories = ['tree', 'plot', 'mapFeature'],
+                simpleSearchKeys = ['species.id', 'mapFeature.geom'],
+                moreSearchFeatureBlacklist = _.union(featureCategories, ['species']),
```

Comment:
Consider moving this `var` definition inside the block where it is used, to simplify things for the reader.